compiler-wasm/Cargo.toml

```diff
@@ -14,6 +14,8 @@ gleam-core = { path = "../compiler-core" }
 console_error_panic_hook = "0"
 serde-wasm-bindgen = "0"
 wasm-bindgen = { version = "0", features = ["serde-serialize"] }
+# See https://docs.rs/getrandom/latest/getrandom/index.html#webassembly-support
+getrandom = { version = "0", features = ["js"] }
```

Comment:
What's this for? No changes have been made to the wasm crate.

Reply:
This dependency is only there to enable the `js` feature on `getrandom` which is a transitive dependency for a few parts of gleam. Without the `js` feature, getrandom would complain whenever its being compiled for `wasm32-unknown-unknown` which now does happen because I bumped the dependencies in hexpm-rust.

I will make an additional comment briefly explaining this and move this to the shared workspace